backstage-create-app doesn't add vital init() piece automatically (#12616)
backstage/backstage: docs/integrations/datadog-rum/installation.md

  #   env: 'staging'
```

> If during [`yarn backstage-create-app`](https://github.com/backstage/backstage/tree/master/packages/create-app) time your [`app-config.yaml`](https://github.com/backstage/backstage/blob/master/app-config.yaml#L5) file did not have this configuration, you have to adjust your [`packages/app/public/index.html`](https://github.com/backstage/backstage/blob/master/packages/app/public/index.html#L64) to include the Datadog RUM `init()` section manually.

Reviewer: ```suggestion
If your [`app-config.yaml`](https://github.com/backstage/backstage/blob/master/app-config.yaml#L5) file does not have this configuration, you may have to adjust your [`packages/app/public/index.html`](https://github.com/backstage/backstage/blob/master/packages/app/public/index.html#L64) to include the Datadog RUM `init()` section manually.
```
